### PR TITLE
Spanish language support

### PR DESCRIPTION
**Pull request by Whoandozco**
I've added full translations for Spanish.

Localized weapon and camouflage categories titles to keep consistency on the languages. I cant speak Swedish, so I added English placeholders for now, on the  `sv-SE.json` file. I don't think doing it for each camo name would be necessary.

image

### CODE REVIEW

File: src/components/LocaleSwitcherComponent.vue [hunk 1]
```diff
@@ -30,7 +30,7 @@ export default {
 	data() {
 		return {
 			open: false,
-			locales: ['en-US', 'sv-SE'],
+			locales: ['en-US', 'sv-SE', 'es-ES'],
```
Comment (carlssonemil) [inline src/components/LocaleSwitcherComponent.vue]: Shouldn't this be `es-CO`? 🤔 
```suggestion
			locales: ['en-US', 'sv-SE', 'es-CO'],
```

**Comment by carlssonemil** [comment]
It looks good! I'll merge it later today, I just need to adjust the font-size on the filter options so they don't look like this 😅:

Screenshot 2022-12-15 at 09 17 45

**Comment by Whoandozco** [comment]
> It looks good! I'll merge it later today, I just need to adjust the font-size on the filter options so they don't look like this 😅:
> 
> Screenshot 2022-12-15 at 09 17 45

Oh! Oops, it didn't look like that for me. I can adjust it if you want.

We could probably remove the wording on the layout as well, maybe in Spanish, "cuadrícula" is a bit too long. Keep the tooltip only?

Thanks for the review! I'll definitely try and contribute some more things, love the app. Great work!

**Comment by carlssonemil** [comment]
> Oh! Oops, it didn't look like that for me. I can adjust it if you want.

No worries! I'll quickly fix it after work! 😄 

> We could probably remove the wording on the layout as well, maybe in Spanish, "cuadrícula" is a bit too long. Keep the tooltip only?

Hm.. Yeah, that's a good idea. I'll look into it!

**Pull request merged**